src/screens/onboarding/onboarding.tsx

```diff
@@ -104,6 +110,7 @@ export class OnboardingScreenComponent extends React.Component<IProps & IReduxPr
         );
     }
     public async onPressGenerateWallet() {
+        this.props.openLoadingModal();
         const password = await hash('000000');
```

Comment:
practic cum e aici, afisezi modalu, "ceri parola", si continui cu modalu

Reply:
Pe generate wallet nu se cere parola, se foloseste `000000`, e doar for developers

Reply:
@NovacAndrei @georgeburduhos as this is a public repo, let's use English as our default language :)

Reply:
Yes, sure :)